packages/lab/src/multiline-input/MultilineInput.tsx

```diff
@@ -0,0 +1,215 @@
+import { clsx } from "clsx";
+import {
+  ChangeEvent,
+  ComponentPropsWithoutRef,
+  FocusEvent,
+  forwardRef,
+  ReactNode,
+  Ref,
+  TextareaHTMLAttributes,
+  useState,
+} from "react";
+import { makePrefixer, StatusAdornment, useControlled } from "@salt-ds/core";
+import { useComponentCssInjection } from "@salt-ds/styles";
+import { useWindow } from "@salt-ds/window";
+
+import multilineInputCss from "./MultilineInput.css";
+
+const withBaseName = makePrefixer("saltMultilineInput");
+
+export interface MultilineInputProps
+  extends Omit<ComponentPropsWithoutRef<"div">, "defaultValue">,
+    Pick<
+      ComponentPropsWithoutRef<"input">,
+      "disabled" | "value" | "defaultValue" | "placeholder"
+    > {
+  /**
+   * The marker to use in an empty read only Input.
+   * Use `''` to disable this feature. Defaults to '—'.
+   */
+  emptyReadOnlyMarker?: string;
+  /**
+   * End adornment component
+   */
+  endAdornment?: ReactNode;
+  /**
+   * Styling variant with full border. Defaults to false
+   */
+  fullBorder?: boolean;
+  /**
+   * [Attributes](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/textarea#Attributes) applied to the `textarea` element.
+   */
+  textAreaProps?: TextareaHTMLAttributes<HTMLTextAreaElement>;
+  /**
+   * Optional ref for the textarea component
+   */
+  textAreaRef?: Ref<HTMLTextAreaElement>;
+  /**
+   * If `true`, the component is read only.
+   */
+  readOnly?: boolean;
+  /**
+   * Start adornment component
+   */
+  startAdornment?: ReactNode;
+  /**
+   * Alignment of text within container. Defaults to "left"
+   */
+  textAlign?: "left" | "center" | "right";
+  /**
+   * Validation status.
+   */
+  validationStatus?: "error" | "warning" | "success";
+  /**
+   * Styling variant. Defaults to "primary".
+   */
+  variant?: "primary" | "secondary";
+  /**
+   * Number of rows. Defaults to 4
+   */
+  rows?: number;
+}
+
+export const MultilineInput = forwardRef<HTMLDivElement, MultilineInputProps>(
+  function MultilineInput(
+    {
+      "aria-activedescendant": ariaActiveDescendant,
+      "aria-expanded": ariaExpanded,
+      "aria-owns": ariaOwns,
+      className: classNameProp,
+      disabled,
+      emptyReadOnlyMarker = "—",
+      endAdornment,
+      fullBorder = false,
+      id,
+      textAreaProps = {},
+      textAreaRef,
+      placeholder,
+      readOnly,
+      role,
+      rows = 3,
```

Comment:
docs indicated the default is 4, not sure which is the correct one 4 or 3

Reply:
good spot thank you! it changed from 4 to 3